SMFloatingLabelTextField/Classes/SMFloatingLabelTextField.m

```diff
@@ -52,8 +52,9 @@ - (void)setup {
 - (void)setupInitialInspectableAttributesValues {
     self.floatingLabelPassiveColor = [UIColor lightGrayColor];
     self.floatingLabelActiveColor = [UIColor blueColor];
-    self.floatingLabelFont = [UIFont systemFontOfSize:12.0];
+    self.floatingLabelFont = [UIFont preferredFontForTextStyle:UIFontTextStyleFootnote];
```

Comment:
Hey Shebuka, thanks for PR. Can you explain why should we use `UIFontTextStyleFootnote` ?